### DISCUSSION TITLE
OTA sketch version is not working as announced?

### DISCUSSION THREAD

**Discussion by danilkorotkov** [HomeSpan/HomeSpan#55] [Q&A] [answered]
How to get the sketch version remotely?
> ... as part of its HAP MDNS broadcast. ... allows you to identify the version of a sketch for a device that is updated via OTA, rather than connected to a computer

![Снимок экрана 2021-03-02 в 10 03 49](https://user-images.githubusercontent.com/25537053/109610972-9735b400-7b3e-11eb-9c35-a2fa9683667e.png)
![Снимок экрана 2021-03-02 в 10 04 01](https://user-images.githubusercontent.com/25537053/109610992-9dc42b80-7b3e-11eb-8d0c-ab8a1c30e4fa.png)
![Снимок экрана 2021-03-02 в 10 04 20](https://user-images.githubusercontent.com/25537053/109611017-a9afed80-7b3e-11eb-9b98-586ce61330e6.png)

**Comment by HomeSpan** [comment, marked as answer]
See:  https://github.com/HomeSpan/HomeSpan/blob/master/docs/FAQ.md#how-can-i-read-homespans-mdns-broadcast-mentioned-in-the-ota-documentation

**Reply by danilkorotkov** [reply]
Thank you)
